evals/registry/evals/speech_translation_tasks.yaml

```diff
@@ -0,0 +1,1114 @@
+### Audio translation tasks (X to EN)
+audio-translate-ar_en:
```

Comment:
is it better to include dataset name to the task, e.g., `audio-translate-covost-ar_en`, as we may add additional datasets to evaluation.